docs/datatypes/core/runtime.md

```diff
@@ -73,14 +64,30 @@ def myAppLogic =
 
 Now we can run that effect using `run` entry point:
 
-```scala mdoc:silent
-object MyApp extends zio.App {
-  final def run(args: List[String]) =
-    myAppLogic.exitCode
+```scala mdoc:compile-only
+object MyApp extends zio.ZIOAppDefault {
+  def run = myAppLogic
+}
+```
+
+ZIO has a service that contains command-line arguments of an application called `ZIOAppArgs`. We can access command-line arguments using `ZIO.service[ZIOAppArgs]` :
+
+```scala mdoc:compile-only
+
+import zio._
+object HelloApp extends ZIOAppDefault {
+  def run = for {
+    args <- ZIO.service[ZIOAppArgs].map(_.args)
```

Comment:
As a feedback, an optional helper method for obtaining arguments would be useful.
cc @adamgfraser

Reply:
It already exists inside `ZIOppDefault` and is called `args`, but maybe it should be called `getArgs`. Probably we should replicated it into the companion object of `ZIOAppArgs` for consistency.